.gitignore

```diff
@@ -25,3 +25,4 @@ myxql-*.tar
 /errmsg-utf8.txt
 /xref_graph.*
 /config
+.elixir_ls
```

Comment:
could you revert this change? We try to keep such editor-specific changes out and instead ask folks to make them in their respective ~/.gitignore_global.

Reply:
Done